README.md

```diff
@@ -278,14 +279,18 @@ network={
 	psk="The password of your WiFi network"
 }
 ```
-* Move the microSD card from your workstation to the Raspberry-Pi and start the device
+The RaspberryPi will get an IP from the DHCP of your network.
 
-Then execute the following steps after obtaining an SSH connection
-to your Raspberry-Pi.
+* To have the changes written corectly, before the umount of the directories "rootfs" and "boot" type "sync" in a command line.
```

Comment:
Please change "corectly" to "correctly"

Reply:
`sync` is a Unix-dependent step (and it should be `unmount`).  Other OS's have different methods, so write a generic description.